spec/support/schools/partnership_helper.rb

```diff
@@ -0,0 +1,11 @@
+module Schools
+  module PartnershipHelper
+    def create_school_partnership_with_lead_provider
+      @contract_period = FactoryBot.create(:contract_period, :current)
+      @lead_provider = FactoryBot.create(:lead_provider, name: "Xavier's School for Gifted Youngsters")
+      @active_lead_provider = FactoryBot.create(:active_lead_provider, contract_period: @contract_period, lead_provider: @lead_provider)
+      @lead_provider_delivery_partnership = FactoryBot.create(:lead_provider_delivery_partnership, active_lead_provider: @active_lead_provider)
+      @school_partnership = FactoryBot.create(:school_partnership, school: @school, lead_provider_delivery_partnership: @lead_provider_delivery_partnership)
```

Comment:
Think we should pass `school` in instead of relying on the instance variable to be set?